### ISSUE TITLE
Supply Chain Management section doesn't focus on the "management"

### ISSUE THREAD

**Issue by ThunderSon** [OWASP/www-project-application-security-awareness-campaigns#1]
As a small improvement, I believe the supply chain management section should look into the "management" of the supply chain, more than the continuous scanning that is mentioned in the implementation phase. This section _should_ focus more around the inventory of applications and services that are deployed/being-used/etc (*BOMs are the direct example here) and the ability to go through them to understand what you have, and what you can filter whenever a need comes up (you need to search for a CVE, you need to understand how many apps use a specific library, etc.)

**Comment by BrettCrawley** [comment]
@ThunderSon 
Agreed, I will add this to the poster, the inventory is a fundamental step that should be performed at the very start to understand your existing real-estate. Performing the scanning upfront, how I describe in the poster, will also help detect projects that need adding to the inventory if engineers are forced to proxy their package managers through the scanner. That way, nothing slips through the net.

"An inventory of your software is fundamental. It allows AppSec to track projects through the SDLC and react to Threat Intelligence with information from the SBOM at hand."

What do you think? Would that be enough, considering the space on the poster it's difficult to fit everything in?

If you agree then I'll update the poster either today or in the next few days.

**Comment by ThunderSon** [comment]
The thing is, the scanner is integrated with the management solution. It has to be "implemented" and then it's continuous scanning happening, which is where you're aiming at. I'd say removing that piece would give you more space to actually talk more about the management section.

The line you provided is very minimal, and SBOM is injected out of the blue.
How about we go off from something to this:
> All applications rely and use on components, whether they are open source or proprietary. In order to secure these parts, having BOMs digested by inventory tooling will help you react to any vulnerabilities that arise in those components, and have full control over what you're building and deploying.

cc @stevespringett - he's one of the best to give a recommendation on this :)

**Comment by BrettCrawley** [comment]
I want to avoid removing that piece though because so many people seem to forget that the dev environment is still a point of compromise. It was my aim to make that apparent.

I’m trying to cover both the pro-active and not just the reactive aspect. The phrase above gives the impression of being predominantly reactive. 

I’m trying to give equal weight to both because if we continue as an industry to ignore the proactive, we will just end up with an ever growing mountain of tech debt.

So I think we need some middle ground between the two.

**Comment by stevespringett** [comment]
OWASP SCVS may have some guidance this project could potentially use.

https://owasp-scvs.gitbook.io/scvs/

> I want to avoid removing that piece though because so many people seem to forget that the dev environment is still a point of compromise. It was my aim to make that apparent.

Development tools are covered by SCVS 3.16. So from an awareness perspective, management should be aware this is an active attack vector.
https://owasp-scvs.gitbook.io/scvs/v3-build-environment
